src/game/objects/creatures/rat.c

```diff
@@ -49,6 +49,15 @@ typedef enum {
 
 static BITE_INFO m_RatBite = { 0, -11, 108, 3 };
 
+static const HYBRID_INFO m_RatInfo = {
+    .land.id = O_RAT,
+    .land.active_anim = RAT_EMPTY,
+    .land.death_anim = RAT_DIE_ANIM,
+    .land.death_state = RAT_DEATH,
+    .water.id = O_VOLE,
+    .water.active_anim = VOLE_EMPTY,
+};
```

Comment:
Why does the water state not need a death anim/state?

Reply:
We never switch to the water death state in the environment checks, so it's not needed. These are the only possible scenarios:
- Killed in water (normal control uses water death anim)
  - Room drained - switch to land death
  - Room re-flooded - remain in land death
- Killed on land
  - Room flooded - remain in land death
  - Room re-drained - remain in land death

The only difference is for crocodiles - if they're killed in water and that room is then drained, but there is still another water room below, they'll remain in the water anim death state.